### PR TITLE
Create TomPawlak.org.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

File: src/chrome/content/rules/TomPawlak.org.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!--
+	^ does not exist.
+-->
+
+<ruleset name="TomPawlak.org">
```
Comment (J0WI) [inline src/chrome/content/rules/TomPawlak.org.xml]: `www.tompawlak.org`

File: src/chrome/content/rules/TomPawlak.org.xml [hunk 2]
```diff
@@ -1,8 +1,6 @@
-<!--
-	^ does not exist.
--->
-
 <ruleset name="TomPawlak.org">
+	<target host="tompawlak.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/TomPawlak.org.xml]: ^ does not exist.

Comment (koops76) [inline src/chrome/content/rules/TomPawlak.org.xml]: Does it exist over HTTP?

**Comment by koops76** [comment]
@J0WI `^` is not accessible either over HTTP or HTTPS. `www` is an empty Heroku app. Should we secure it anyway? I also added a rule which redirects from `^` to `www` since many people do not type `www` in the URL (like me).

**Comment by J0WI** [comment]
We don't add targets that do not exist. Please remove `^` completely.
Since `www` returns a 404 you can also just keep a comment for it.

**Comment by koops76** [comment]
@J0WI Done.

**Pull request merged**